src/H5ESevent.c

```diff
@@ -185,6 +185,14 @@ H5ES__event_completed(H5ES_event_t *ev, H5ES_event_list_t *el)
     /* Sanity check */
     HDassert(ev);
 
+    /*
+        HDfprintf(stderr,
+                  "%s: releasing event for '%s' (count: %llu, timestamp: %llu), with args '%s', in app source
+       " "file '%s', function '%s', and line %u\n", FUNC, ev->op_info.api_name, (unsigned long
+       long)ev->op_info.op_ins_count, (unsigned long long)ev->op_info.op_ins_ts, ev->op_info.api_args,
+       ev->op_info.app_file_name, ev->op_info.app_func_name, ev->op_info.app_line_num);
```

Comment:
Would it be possible to loop this into HDF5's debugging capabilities? Not much reason to leave a commented out fprintf statement in the library.

Reply:
There's already similar info available in the error info for failed async tasks, I'll take this debugging printf out.